New Anarch Bar, Base, and Garage Addition And Radios (And Knuckles)

SoreYew opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

I got some donated assets from Ember16, and had to reap what I had sowed with my poor coding practices, so here it is on a new PR from #209.
Features: 

HQ Changes:
- A coat check, for enforcing decorum, when it suits the Anarchs
- A more formal stage for music, spoken word, impromptu Praxis meetings, etc 
- a private 'section' for hiding in plain sight
- janitor closet
- A kitchen with food (and blood) for anarchs to eat. 
- A lockerroom with extra clothes, jackets, boots, and washing machines. 
- Moved the Anarch cars all to clearly anarch owned areas, to make it clear they belong to the faction. Did you know anarchs have had roundstart cars since VTM13?
- Anarchs can lock the doors if they so choose, to force a flow of traffic or lock down. Doors start unlocked if they should, like public doors
- Jail cells, for those who cross the Baron
- Showers and a rec room for the Anarchs. Hold meetings, get high, sleep in the corner. 
- Armory gains some .45 handguns, snubnoses, and stakes.

Garage addition:
- Garage with shutters that require anarch ID to open, holds some faction cars
- has a medical setup in the back, and some medical supplies. 

Code Additions:
8 new decals for wall fliers, thank you again Ember16 for donating them to the codebase. 
![image](https://github.com/user-attachments/assets/21b4333f-8137-465a-90b0-1ea4cbada632)
4 new posters with old versions, thank you Ember16! 
![image](https://github.com/user-attachments/assets/20409720-40ac-4dcc-a629-da80bf7fa81b)


## Why It's Good For The Game

New bar for the new decade, plus a garage. The anarchs should have a bunch of little things, and this is a start for that. Plus some flavorful art assets. 

## Testing Photographs and Procedure
![image](https://github.com/user-attachments/assets/6d7626d4-c8dc-4320-9b58-6bd3d35abd7c)
![image](https://github.com/user-attachments/assets/4b352c37-56db-4bae-a291-736f37fbcd1e)
![image](https://github.com/user-attachments/assets/e8997fe9-d728-444b-ac33-fa12a6e73faa)
![image](https://github.com/user-attachments/assets/816db305-80f8-4d57-a62e-f1b32883505f)

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and its effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
add: Redesigned the Anarch Bar
add: Garage building east of the bar, west of the hospital
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

AllCorruptingWyrm commented:
very nice!

ShadeAware commented:
![d81cf9eonwve1](https://github.com/user-attachments/assets/a5da0511-436f-4277-9c02-6b0a2b779164)

Marshmellow105 commented:
all of your northfacing lightbulbs are done wrong, they need to be the variant that is pixelshifted up (which I think is preset when you're changing the dirs)

as is, they're inside the wall.

![image](https://github.com/user-attachments/assets/9ea7826d-7b68-4b32-bd68-e3c03c887f2e)


you also have an abundance in lighting, like in the above screenshot, there is three lightbulbs in one office. this is present across the remap by the looks of it

your curtains here would also look better pixelshifted upwards. I dont know the exact number you want to move them because its something I am also figuring out, but worth noting.

![image](https://github.com/user-attachments/assets/2d550613-7f9b-4902-828b-510f386e8f24)

because of how multiz displays lower levels this looks kinda wack, I would remove the fake stairs, replace them with flooring, and instead cut the flooring behind the staircase back to the red line here

![image](https://github.com/user-attachments/assets/6949415a-1f8d-49e3-8f93-445aed05fba6)


you really nailed the vibe here with the help of the posters, this area and the garage are the two areas of this PR that I really like

![image](https://github.com/user-attachments/assets/5eb24c74-0c0f-4b71-a6fe-dff01fc67d4c)

SoreYew commented:
![dreamseeker_hY7gcHjuNk](https://github.com/user-attachments/assets/cb559004-b4a2-48a7-b53a-9a8319e375de)
![image](https://github.com/user-attachments/assets/032f76f1-2080-46bc-b66a-e71aa0c5121d)
posters working as intended.


Made the whole place dimmer. Took suggestions for the staircase. Changed file organization. Added movie posters. Fixed roofs, etc. Gonna work on pixelshifting the curtains and metal bars, and whatever else people find.

SoreYew commented:
![dreamseeker_UV5ImsUqZi](https://github.com/user-attachments/assets/3aacf534-7faf-45b8-bd86-6b6ca434c683)
![StrongDMM_3EJa4JfDuP](https://github.com/user-attachments/assets/c45770c9-4dc5-4f0f-bd1f-e3b264d2a399)
fixed the bars issue, added more filler upstairs. Curtains i can't figure out, so they're the same.

Kevak commented:
I've heard reports part of it is unroofed

Kevak commented:
Good good,

SoreYew commented:
![image](https://github.com/user-attachments/assets/cef96d82-a03d-46ba-9333-2bc050257f56)
Added a nice window so people can see if anybody is actually attempting to use the building for something.  Other than that, complaints addressed.

SoreYew commented:
![dreamseeker_8wAQEJbY6U](https://github.com/user-attachments/assets/1e55e146-21a9-43a5-8cd9-43463bfaa8c0)
![dreamseeker_dmTjv5ty95](https://github.com/user-attachments/assets/353f8b97-37a9-4004-8616-b599e9fa84f2)
![StrongDMM_0Ulcq5r5WC](https://github.com/user-attachments/assets/241926e8-b426-4c4c-9510-68b9602720cb)
![StrongDMM_0rrYCxptOV](https://github.com/user-attachments/assets/7dfb1b77-2f88-4865-b26a-0daef2345602)
![StrongDMM_1JLHmBubh4](https://github.com/user-attachments/assets/95293098-3dba-4a01-ada5-4718e1474d6e)
![StrongDMM_pd2l0jNTqW](https://github.com/user-attachments/assets/577a522b-228b-4374-8129-007a85bce7c7)
![StrongDMM_zPO4VZPB1a](https://github.com/user-attachments/assets/6670e712-83a0-498d-a00e-9a21963e3acc)
![StrongDMM_3r6SDphRgV](https://github.com/user-attachments/assets/86f7dbdd-acff-408c-8d46-b9fb69f5b5f3)
